src/tools/fuzzing/fuzzing.cpp

```diff
@@ -2621,15 +2633,29 @@ Expression* TranslateToFuzzReader::makeBasicRef(Type type) {
       // Use a local static to avoid the expense of canonicalizing a new type
       // every time.
       static HeapType trivialStruct = HeapType(Struct());
-      return builder.makeStructNew(trivialStruct, std::vector<Expression*>{});
+      static HeapType sharedTrivialStruct = []() {
+        TypeBuilder builder(1);
+        builder[0] = Struct{};
+        builder[0].setShared();
+        return (*builder.build())[0];
+      }();
+      auto ht = share == Shared ? sharedTrivialStruct : trivialStruct;
```

Comment:
```suggestion
      static HeapType makeSharedTrivialStruct = []() {
        TypeBuilder builder(1);
        builder[0] = Struct{};
        builder[0].setShared();
        return (*builder.build())[0];
      }();
      auto ht = share == Shared ? makeSharedTrivialStruct() : trivialStruct;
```

Reply:
(also below, if this makes sense)

Reply:
Note the `()` after the lambda expression making this an IIFE meant to be executed once when the static local is initialized.

Reply:
Thanks, I missed the `static`.